data/czech-cs.txt

```diff
@@ -263,4 +263,4 @@ country: cz
 27 f john_evangelist : Sv. Jana, apoštola a evangelisty
 28 f R innocents : Svatých Mláďátek, mučedníků
 29 R becket : Sv. Tomáše Becketa, biskupa a mučedníka
-31 sylvester_i : Sv. Silvestra I. papeže
```

Comment:
Looks like you (or perhaps your editor automatically) removed the newline at the end of the last line. Please remove this change, we do want the newline there.

Reply:
Oh, sorry. Yes, I have configured my global `.editorconfig` to remove trailing lines and I didn’t check the changes I was about to commit. My bad.